src/transformers/models/falcon/configuration_falcon.py

```diff
@@ -72,6 +72,19 @@ class FalconConfig(PretrainedConfig):
             instead, as in the original Transformer architecture. Ignored when `new_decoder_architecture` is `True`.
         bias (`bool`, *optional*, defaults to `False`):
             Whether to use bias on Linear layers.
+        max_position_embeddings (`int`, *optional*, defaults to 2048):
+            The maximum sequence length that this model might ever be used with, when `alibi` is `False`. Pretrained
+            Falcon models with RoPE support up to 2048 tokens.
+        rope_theta (`float`, *optional*, defaults to 10000.0):
+            The base period of the RoPE embeddings.
```

Comment:
Why isn't this part of the rope scaling dict?

Reply:
Technically this isn't a rope scaling parameter, it is a constant used to compute the embeddings. As such, I agree with the original decision to keep it separate, in the CodeLlama PR :)

Coincidently, it can also be used for scaling (i.e. by increasing it then fine-tuning it).